CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 ## Added
 - Added this changelog.
 
+## Changed

```

Comment:
I was hoping for an "Added PuTTY private key v3 support" entry. 😄 

Internal cleanups aren't really worth mentioning here, IMHO.

Reply:
Ah my mistake.

Reply:
I removed my change and added your suggestion.